tests/api.c

```diff
@@ -4300,6 +4338,10 @@ static void test_wolfSSL_BN(void)
     value[0] = 0x05;
     AssertNotNull(BN_bin2bn(value, sizeof(value), c));
 
+    /* BN_div a/b */
+    AssertIntEQ(BN_div(d, a, b, NULL, NULL), SSL_FAILURE);
```

Comment:
Please add unit tests which verify the output of BN_div() is correct, checking against a known value.